docs/api/event-brokers.rst

```diff
@@ -29,35 +43,28 @@ tracker looks like this:
 The ``event`` field takes the event's ``type_name`` (for more on event
```

Comment:
I think this json format is old as well, does it need some metadata?

Reply:
yes, `bot` events always have `data` and `metadata`